pkg/minikube/cruntime/docker.go

```diff
@@ -365,6 +365,25 @@ func dockerImagesPreloaded(runner command.Runner, images []string) bool {
 	return true
 }
 
+// ForceSystemdCgroupManager forces docker to use systemd as cgroup manager, as recommended in k8s docs:
+// https://kubernetes.io/docs/setup/production-environment/container-runtimes/#docker
+func (r *Docker) ForceSystemdCgroupManager() error {
```

Comment:
how about we do this during the installation ? in kicbase ?
so we don't have to restart docker and also copy config files?

Reply:
https://github.com/kubernetes/minikube/blob/master/hack/images/kicbase.Dockerfile#L12

Reply:
cool, done